pronto.gemspec

```diff
@@ -54,5 +54,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency('rspec-its', '~> 1.2')
   s.add_development_dependency('rspec-expectations', '~> 3.4')
   s.add_development_dependency('rubocop', '~> 0.58')
-  s.add_development_dependency('simplecov', '~> 0.14')
+  # Enforces a rubocop version that is supported by CodeClimate, they support 0.17.x or >= 0.20
```

Comment:
Can you add a link on where this is documented? Also, your comment mentions `rubocop` but the dependency is for `simplecov`.

Reply:
Sorry about that, fixed.